schemas/deviceToCloud/device/device.json

```diff
@@ -54,9 +54,9 @@
                     "type": "integer"
                 },
                 "mccmnc": {
-                    "type": "string",
+                    "type": "integer",
```

Comment:
https://github.com/nrfconnect/sdk-nrf/blob/3797fac81db0b317d14abded836b3ebc3220979c/lib/modem_info/modem_info_params.c#L71

this has been a string for a long time.
ATv2 has its own implementation, which is a number.
i have no preference, other than that all code in NCS is aligned.